### PR TITLE
Added Links for German and Esperanto

### PR DESCRIPTION
**Pull request by stefangrotz**

### CODE REVIEW

**Comment by stefangrotz** [comment]
See also #3156 for the missing language codes

File: docs/COMMUNITIES.md [hunk 1]
```diff
@@ -15,8 +15,14 @@ Many language communities are self organising and have their contact channels on
...
    * [Discord](https://discord.com/invite/5ZdTHsN8Mn)
    * [Matrix](https://app.element.io/#/room/!WNiJFJTBpvYqMrDWir:matrix.org)
+* Esperanto 
+   * [Matrix](https://chat.mozilla.org/#/room/#Esperanto:mozilla.org)
+   * [GitHub](https://github.com/parolrekonado)
 * French (`fr`):
    * [Matrix](https://chat.mozilla.org/#/room/#common-voice-fr:mozilla.org)
+* German
+   * [Matrix](https://chat.mozilla.org/#/room/#common-voice-de:mozilla.org)
+   * [Discord](https://discourse.mozilla.org/c/voice/de/289)
```
Comment (MichaelKohler) [inline docs/COMMUNITIES.md:25]: Discourse ;)

File: docs/COMMUNITIES.md [hunk 2]
```diff
@@ -15,8 +15,14 @@ Many language communities are self organising and have their contact channels on
    * [Web](https://dhivehi.ai/)
    * [Discord](https://discord.com/invite/5ZdTHsN8Mn)
    * [Matrix](https://app.element.io/#/room/!WNiJFJTBpvYqMrDWir:matrix.org)
+* Esperanto
```
Comment (MichaelKohler) [inline docs/COMMUNITIES.md:18]: Let's also add the language codes for both languages

**Review by MichaelKohler — CHANGES_REQUESTED** [review]

File: docs/COMMUNITIES.md (hunk 2 above)
Comment (MichaelKohler) [reply docs/COMMUNITIES.md:18]: Oh, I see the other PR now.

**Comment by MichaelKohler** [comment]
Closing this as https://github.com/common-voice/common-voice/pull/3156/files covers the fixes as well

**Pull request closed without merging**